### PR TITLE
feat(*): Added limit to number of notifications to be displayed at once.

### PR DESCRIPTION
**Pull request by chrisjshelby**
Needed to limit the number of notifications shown at once.

### CODE REVIEW

File: src/notification-renderer.js [hunk 1]
```diff
@@ -31,7 +32,13 @@ export class NotificationRenderer {
     notificationController.slot.add(notificationController.view);
 
     notificationController.showNotification = () => {
+            
       this.notificationControllers.push(notificationController);
+     
+      if (this.notificationControllers.length >= (globalSettings.limit + 1)) {
```
Comment (MarcScheib) [inline src/notification-renderer.js]: Can you switch from globalSettings.limit to settings.limit? It is configurable then.

**Comment by MarcScheib** [comment]
HI @chrisjshelby,
nice feature. Can you have look at my comment? Although, can you lint the code again, travis gives the following lint errors:

  34:53  error  Block must not be padded by blank lines  padded-blocks
  35:1   error  Trailing spaces not allowed              no-trailing-spaces
  37:1   error  Trailing spaces not allowed              no-trailing-spaces
  41:1   error  Trailing spaces not allowed              no-trailing-spaces

File: src/notification-renderer.js (hunk 1 above)
Comment (chrisjshelby) [inline src/notification-renderer.js]: Sure thing... I'll make that switch...

**Comment by MarcScheib** [comment]
Added version 0.3.0 with your change. Thanks for your contribution @chrisjshelby :+1:

**Pull request merged**